sdk/tables/data-tables/test/public/tableclient.spec.ts

```diff
@@ -401,7 +401,10 @@ authModes.forEach((authMode) => {
       });
 
       it("should createEntity with primitive int and float without automatic type conversion", async () => {
-        type TestType = { integerNumber: number; floatingPointNumber: number };
+        type TestType = {
```

Comment:
Should we add a boolean to this object so we have a recorded test case with a boolean property using `disableTypeConversion: true`?  From the description of the PR it seems like if we had that initially we would have discovered at least part of the bug?

Reply:
Yeah that's a great point, I'll update the recordings

Reply:
@ellismg updated the test and recordings to include a boolean property in the entity. Thanks for bringing this up :smile: